src/commons/api/ca.rs

```diff
@@ -1590,6 +1600,13 @@ impl ChildConnectionStats {
     pub fn new(handle: ChildHandle, last_exchange: Option<ChildExchange>) -> Self {
         ChildConnectionStats { handle, last_exchange }
     }
+
+    pub fn inactive(&self, threshold_hours: i64) -> bool {
```

Comment:
There are probably places like this all over the code that use an integer for some unit of time but I was just reminded by https://rust-lang.github.io/api-guidelines/type-safety.html#c-newtype that an alternate approach could be to use an "hours" type rather than i64 with a name including the term "_hours".